src/components/ToastMessage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+function App() {
```

Comment:
```suggestion
function ToastMessage() {
```

Reply:
Thanks! I made this change in the second branch I just created.